src/vendor_upstream/dom/ReactTouchHandler.js

```diff
@@ -0,0 +1,112 @@
+/**
```

Comment:
For now lets move this directly into src rather than vendor_upstream since there's no upstream for it.